lib/engine.py

```diff
@@ -698,16 +696,29 @@ def table_name(self, name=None, dbname=None):
         return self.opts["table_name"].format(db=dbname, table=name)
 
     def warning(self, warning):
-        new_warning = Warning('%s:%s' % (self.script.shortname, self.table.name), warning)
+        new_warning = Warning('%s:%s' % (
+            self.script.shortname, self.table.name), warning)
         self.warnings.append(new_warning)
 
 
+def load_data(filename, delimiter="\t"):
+    reg = re.compile("\\r\\n|\n|\r")
+    with io.open(filename,  newline='', encoding='latin-1') as dataset_file:
+        for row in csv.reader(dataset_file,  delimiter=delimiter,  quoting=csv.QUOTE_MINIMAL):
+            temp_list = []
+            for fields in row:
+                clean = reg.sub(" ", fields)
+                temp_list.append(clean)
+            yield '{0}'.format(delimiter).join(temp_list)
+
+
 def skip_rows(rows, source):
     """Skip over the header lines by reading them before processing."""
     lines = gen_from_source(source)
     for i in range(rows):
         next(lines)
-    return lines
+    # return lines
+    return source
```

Comment:
I'm confused as to what's going on here. Should we really be returning `source`? Can you explain the choice and document it in the docstring?